pages/projects/[id].tsx

```diff
@@ -143,6 +143,11 @@ const Project: NextPage = (): JSX.Element => {
       // A null after_id means the story was placed last in the list.
     };
 
+    if (destinationState === 'accepted') {
+      delete payload.before_id;
+      delete payload.after_id;
+    }
+
```

Comment:
I think if we add the props conditionally to the constant definition does the same, but we don't mutate the object. 
```js
const payload = {
    current_state: destinationState,
    ...( destinationState !== 'accepted && {
      before_id: stories[destinationState][landingIndex]?.id || null,
      // A null before_id means the story was placed first in the list.
      after_id: stories[destinationState][landingIndex - 1]?.id || null,
      // A null after_id means the story was placed last in the list.
     } : {}
   ),
  };
```